internal/http/webhook.go

```diff
@@ -25,10 +25,28 @@ func newWebhookHandler(cfg *domain.Config, processorSvc *processor.Service) *web
 func (h webhookHandler) Routes(r chi.Router) {
 	r.Get("/trigger", h.run)
 	r.Post("/trigger", h.run)
+	r.Get("/trigger/arr", h.arr)
+	r.Get("/trigger/lists", h.lists)
+	r.Post("/trigger/arr", h.arr)
+	r.Post("/trigger/lists", h.lists)
 }
 
 func (h webhookHandler) run(w http.ResponseWriter, r *http.Request) {
-	if err := h.processorService.Process(false); err != nil {
+	if err := h.processorService.Process("both", false); err != nil {
```

Comment:
You can keep the main `Process` function but skip the type and run the arrs and lists functions after each other, or call them separately in this http handler.